tests/sequence_labelling/preprocess_test.py

```diff
@@ -132,3 +132,31 @@ def test_should_select_features(self):
         features_length = len(preprocessor.features_indices)
         assert features_length == 1
         assert all_close(features_transformed, [[[1], [2], [3]]])
+
+    @pytest.fixture(scope="function")
+    def test_serialize_to_json(self, tmpdir_factory):
+        preprocessor = FeaturesPreprocessor(features_indices=[1])
+        features_batch = [[
+            [FEATURE_VALUE_1, FEATURE_VALUE_2],
+            [FEATURE_VALUE_1, FEATURE_VALUE_3],
+            [FEATURE_VALUE_1, FEATURE_VALUE_4]
+        ]]
+        preprocessor.fit(features_batch)
+        word_preprocessor = WordPreprocessor(feature_preprocessor=preprocessor)
+
+        import tempfile
+        f = tmpdir_factory.mktemp("data").join("serialised.json")
```

Comment:
You could potentially shorten it slightly by using `tmp_path` (I haven't tested the `tmp_path` fixture yet):

```python
from pathlib import Path

def test_serialize_to_json(self, tmp_path: Path):
    # ...
   temp_file = tmp_path / 'serialised.json'
   word_preprocessor.save(file_path=str(temp_file))
```

Reply:
indeed